pint/pint_eval.py

```diff
@@ -9,14 +9,24 @@
 """
 from __future__ import annotations
 
+from io import BytesIO
 import operator
 import token as tokenlib
 import tokenize
 
+try:
+    from uncertainties import ufloat
+    HAS_UNCERTAINTIES = True
+except ImportError:
+    HAS_UNCERTAINTIES = False
+    ufloat = None
+
 from .errors import DefinitionSyntaxError
 
 # For controlling order of operations
 _OP_PRIORITY = {
+    "±": 4,
```

Comment:
This line can go away: we change ± to +/- to avoid gratuitous problems with ERRORTOKEN.